utility/sort/sort_test.go

```diff
@@ -0,0 +1,40 @@
+package sort
+
+import "github.com/stretchr/testify/suite"
+import (
+	"github.com/gojektech/proctor/proctord/jobs/metadata"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+type SortTestSuite struct {
+	suite.Suite
+	sorter *commandSorter
+}
+
+func (s *SortTestSuite) setupTest() {
+	s.sorter = new(commandSorter)
+}
+
+func (s *SortTestSuite) TestSorting() {
+	procOne := metadata.Metadata{
+		Name:        "one",
+		Description: "proc one description",
+	}
+
+	procTwo := metadata.Metadata{
+		Name:        "two",
+		Description: "proc two description",
+	}
+
+	procList := []metadata.Metadata{procTwo, procOne}
```

Comment:
can we have 3 procs to be sorted? just a little more stringent i guess. No need to specify description field.